plugins/typescript/src/core/schemaToTypeAliasDeclaration.ts

```diff
@@ -165,6 +165,9 @@ export const getType = (
         schema.nullable
       );
     case "string":
+      if (schema.format === "binary") {
+        return f.createTypeReferenceNode("Blob");
```

Comment:
I used Blob instead of File to be safe.

Reply:
As a reference:
 https://github.com/cellular/oazapfts/blob/master/src/codegen/generate.ts#L431